wordle-web/src/components/LetterBase.vue

```diff
@@ -1,5 +1,13 @@
 <template>
-  <v-btn :color="props.color">
+  <v-btn 
+     class="charButton"
```

Comment:
By default the buttons will have a shadow when they have some elevation, but I think I'd add this just to be safe
```suggestion
     class="charButton, shadow-5"
```